src/channel.cpp

```diff
@@ -725,6 +728,11 @@ void CChannel::PrepAndSendPacket ( CHighPrioSocket*        pSocket,
                                    const CVector<uint8_t>& vecbyNPacket,
                                    const int               iNPacketLen )
 {
+    if (bIsServer && !bIsIdentified)
```

Comment:
Sadly, this is another two checks for each packet of each client, as far as I understand. I don't see a way to change this though and I can't judge what the performance impact is. I'd say that two comparisons should be cheap, but I lack knowledge about how much this is percentage-wise.

Reply:
I would think compared with all the other per-channel operations (codec, mixing, etc.), it's infinitesimal.